mungegithub/Makefile

```diff
@@ -62,12 +62,12 @@ endif
 # Launch the container on a cluster (with --dry-run).
 # The cluster will likely need a service to get access to the web interface (see service.yaml)
 # The cluster will need a github oauth token (the secret target makes that easy to create)
-deploy: push deployment get-cluster-credentials
```

Comment:
Does this break current users? What's the workaround? cc @foxish @rmmh

Reply:
There is this thread https://github.com/kubernetes/test-infra/pull/4538/files#r141820190 and a follow-up issue https://github.com/kubernetes/test-infra/issues/4814.

Will drop this commit from the PR and keep the change locally.